src/app/cat-filter/cat-filter.component.html

```diff
@@ -0,0 +1,27 @@
+<form [formGroup]="catSelects">
+  <mat-form-field>
+    <mat-select formControlName="category">
+      <mat-option [value]="''"></mat-option>
+      <mat-option
+        *ngFor="let category of (categoriesList$ | async); let i = index"
+        [value]="category.id"
+      >{{ category.name | titlecase }}</mat-option>
+    </mat-select>
+  </mat-form-field>
+  <mat-form-field>
+    <mat-select formControlName="breeds">
+      <mat-option [value]="''"></mat-option>
```

Comment:
do we need value as variable here?

Reply:
it's empty string for searching. If we are searching with breed id, we can't search with category. This stuff works in both ways. So we need an empty value for both options.
If you ask about brackets, so I don't know. Already changed it with regular html attribute.